views/about.ejs

```diff
@@ -3,7 +3,48 @@
 	<%- include('partials/header'); %>
 	<main class="mdl-layout__content">
 		<div class="mdl-typography--text-center clemson-esports">
-			<h1>Clemson Esports</h1>
+			<h1>About Clemson Esports</h1>
 		</div>
+		<div class="mdl-typography--text-center clemson-esports">
+			<p>
+				More than just a competitive eSports Team, Clemson Esports is a community of professional gamers and
+				eSports fans. What began as a group of friends competing online evolved into a leading Esports
+				Team with the support of reputable sponsors and affiliated organizations.
+			</p>
+			<p>
+				While coming from diverse backgrounds, the players of Clemson Esports are driven by common pursuits;
+				a passion for gaming, a thirst for competition, and a desire to share our excitement and experiences.
+			</p>
+			<p>
+				Clemson Esports started out as a small League of Legends club on campus. We have since grown from a
+				hand-full of members to several hundred over the course of a couple of years. We have gone on to field
+				many new games and help expand our roster. Our players play in front of thousands of viewers on stages
+				around the country. Our club has become a home for all gamers on campus, competitive and casual alike.
+				If you love Esports and gaming and want to make friends with fellow students with the same interests,
+				then join the club today.
+			</p>
+		</div>
+		<div class="about-absolute-center"> <!-- This div is for the buttons -->
+			<div class="about-button-container">
+				<a class="mdl-navigation__link" href="rosters">
+					<button class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect clemson-orange">
```

Comment:
Buttons look too small in mobile preview. The webpage needs to support both desktop and mobile devices. There should be a way to change the way the buttons are displayed based on the screen size or device type. (I've done this before in Tailwinds)
![image](https://user-images.githubusercontent.com/14143235/160151094-cd2e195b-368d-475c-a359-9457f5f38c1d.png)

Reply:
New buttons 
![image](https://user-images.githubusercontent.com/24688719/162633873-761376e5-c9e7-43f1-8dd7-1767609d48a4.png)

Reply:
since you say it works fine on your end @Emimendoza, I'm going to push to dev and see how it looks on my iPhone.